en/docs/reference/config-catalog.md

```diff
@@ -921,6 +921,23 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
                             </div>
                         </div>
                     </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="7" type="checkbox" id="_tab_31">
+                <label class="tab-selector" for="_tab_31"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+                        <pre><code class="toml">[[apim.jwt.issuer]]
```

Comment:
`[[ ]]` brackets seem incorrect here.

<img width="796" alt="Screenshot 2020-10-22 at 10 50 48" src="https://user-images.githubusercontent.com/5195851/96828159-bc117080-1454-11eb-9ade-9a2d2086bcce.png">

Reply:
Double brackets are added because we can have multiple such configs

Reply:
Double brackets are added because we can have multiple such configs